app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+	xmlns:app="http://schemas.android.com/apk/res-auto"
+	xmlns:tools="http://schemas.android.com/tools"
+	android:layout_width="match_parent"
+	android:layout_height="match_parent"
+	android:gravity="center"
+	android:orientation="vertical">
+
+	<ImageView
+		android:id="@+id/vLoginLogoCover"
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content"
+		android:background="@drawable/login_cover"
+		android:src="@drawable/login_cover_gradient"
+		app:layout_constraintEnd_toEndOf="parent"
+		app:layout_constraintStart_toStartOf="parent"
+		app:layout_constraintTop_toTopOf="parent" />
+
+	<ImageView
+		android:id="@+id/vLoginWoloxLogo"
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content"
+		android:layout_alignParentStart="true"
+		android:layout_marginTop="@dimen/login_cover_logo_height"
+		android:adjustViewBounds="false"
+		android:gravity="bottom|center_horizontal"
+		android:src="@drawable/wolox_logo_large"
+		app:layout_constraintEnd_toEndOf="parent"
+		app:layout_constraintHorizontal_bias="0.0"
```

Comment:
I think this was add automatically when using the graph editor. You can delete the `bias`, unless you really use it, in that case leave it